config/formats.ts

```diff
@@ -997,18 +996,25 @@ export const Formats: FormatList = [
 				return priority + forte.priority;
 			}
 		},
+		onModifyTypePriority: 1,
+		onModifyType(move, pokemon, target) {
+			const forte = pokemon.m.forte;
+			if (move.category !== 'Status' && forte) {
+				this.singleEvent('ModifyType', forte, null, pokemon, target, move, move);
+			}
+		},
```

Comment:
This is how I had it originally and it didn't work iirc. Can you confirm it works this time?